app/build.gradle

```diff
@@ -16,10 +16,14 @@ android {
             proguardFiles getDefaultProguardFile('proguard-android-optimize.txt'), 'proguard-rules.pro'
         }
     }
+    compileOptions {
+        sourceCompatibility JavaVersion.VERSION_1_8
+        targetCompatibility JavaVersion.VERSION_1_8
```

Comment:
@Scrollend723 is it supported by android 4?

Reply:
https://developer.android.com/studio/write/java8-support